packages/dd-trace/test/opentelemetry/protobuf.spec.js

```diff
@@ -0,0 +1,205 @@
+'use strict'
+
+const { expect } = require('chai')
+const { describe, it, beforeEach } = require('tap').mocha
+const OtlpTransformer = require('../../src/opentelemetry/logs/otlp_transformer')
+const { SeverityNumber } = require('@opentelemetry/api-logs')
+
+describe('OTLP Protobuf Serialization', () => {
+  let transformer
+
+  beforeEach(() => {
+    transformer = new OtlpTransformer({
+      resource: {
+        attributes: {
+          'service.name': 'test-service',
+          'service.version': '1.0.0',
+          'deployment.environment': 'test'
+        }
+      }
+    })
+  })
+
+  it('should serialize log records to protobuf format', () => {
+    const logRecords = [
+      {
+        timestamp: Date.now() * 1000000,
+        severityNumber: SeverityNumber.INFO,
+        severityText: 'INFO',
+        body: 'Test log message',
+        attributes: {
+          'test.attribute': 'test-value',
+          'user.id': '12345'
+        }
+      }
+    ]
+
+    const result = transformer.transformLogRecords(logRecords)
+
+    expect(result).to.be.instanceOf(Buffer)
+    expect(result.length).to.be.greaterThan(0)
+  })
+
+  it('should handle different severity levels', () => {
+    const logRecords = [
+      {
+        severityNumber: SeverityNumber.DEBUG,
+        severityText: 'DEBUG',
+        body: 'Debug message'
+      },
+      {
+        severityNumber: SeverityNumber.ERROR,
+        severityText: 'ERROR',
+        body: 'Error message'
+      },
+      {
+        severityNumber: SeverityNumber.FATAL,
+        severityText: 'FATAL',
+        body: 'Fatal message'
+      }
+    ]
+
+    const result = transformer.transformLogRecords(logRecords)
+
+    expect(result).to.be.instanceOf(Buffer)
+    expect(result.length).to.be.greaterThan(0)
+  })
+
+  it('should handle trace and span IDs', () => {
+    const logRecords = [
+      {
+        body: 'Test message with trace context',
+        traceId: '12345678901234567890123456789012',
+        spanId: '1234567890123456'
+      }
+    ]
+
+    const result = transformer.transformLogRecords(logRecords)
+
+    expect(result).to.be.instanceOf(Buffer)
+    expect(result.length).to.be.greaterThan(0)
+  })
+
+  it('should handle different body types', () => {
+    const logRecords = [
+      {
+        body: 'String message',
+        severityNumber: SeverityNumber.INFO
+      },
+      {
+        body: 42,
+        severityNumber: SeverityNumber.INFO
+      },
+      {
+        body: true,
+        severityNumber: SeverityNumber.INFO
+      },
+      {
+        body: { nested: { value: 'object' } },
+        severityNumber: SeverityNumber.INFO
+      }
+    ]
+
+    const result = transformer.transformLogRecords(logRecords)
+
+    expect(result).to.be.instanceOf(Buffer)
+    expect(result.length).to.be.greaterThan(0)
```

Comment:
Or logRecords length
```suggestion
    expect(result.length).to.be.greaterThan(4)
```

Reply:
Removed test